### DISCUSSION TITLE
Notebook and package for view scenes of Sentinel and Landsat

### DISCUSSION THREAD

**Discussion by lmotta** [microsoft/PlanetaryComputer#185] [Show and tell]
Hi Guys,

We are testing the Planetary Computer  for monitoring and combat of fires in Brazil.

This is the first contribuition for this community, and I hope this work is useful for search and validate scenes of Landsat and Sentinel.

More information is avaliable this [wiki](https://github.com/lmotta/ms_planetary_scenes/wiki).

Luiz Motta,
Enviroment Analyst

Division of Monitorament and Combat - DMC
National Center for Wildfire Prevention and Suppression - Prevfogo
Brazilian Institute for the Environment and Renewable Natural Resources - [Ibama](https://www.gov.br/ibama/pt-br)

**Comment by TomAugspurger** [comment]
Thanks for sharing! Just an FYI, you *might* be able to use nbgitpuller to automatically download and open the notebook you want. https://hub.jupyter.org/nbgitpuller/link is a tool for generating the link. We use that for our "Launch in Hub" links on the example notebooks in our data catalog (e.g. `https://pccompute.westeurope.cloudapp.azure.com/compute/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPlanetaryComputerExamples&urlpath=lab%2Fworkspaces%2Fgoes-goes-cmi-example%2Ftree%2FPlanetaryComputerExamples%2Fdatasets%2Fgoes%2Fgoes-cmi-example.ipynb&branch=main`)


Let us know if there's anything on the STAC API / tiler side that could be improved.

**Comment by lmotta** [comment]
Thanks @TomAugspurger ,

I don't know if the 'nbgitpuller' will work with import package.
The my notebook import the 'scenes_package'.

Of course if I can share by link, it is the best choice.

If you can, please, test the notebook with the interval of dates large (~5 months). In Brazil, I found  some scenes with corrupted TIFF (Landsat collections).
You can see the table with errors
![image](https://user-images.githubusercontent.com/193559/218877797-766ee964-cd66-4e9e-9887-79853ad0f5b8.png)

**Reply by TomAugspurger** [reply]
Most likely you're running into https://github.com/microsoft/PlanetaryComputer/discussions/101. Follow that thread for updates (progress is a bit slow since some of the original files have apparently been deleted from the USGS servers and replaced with reprocessed scenes)
